Source/Core/ElementStyle.cpp

```diff
@@ -784,6 +784,10 @@ PropertyIdSet ElementStyle::ComputeValues(Style::ComputedValues& values, const S
 			// (font-size computed above)
 			dirty_font_face_handle = true;
 			break;
+		case PropertyId::LetterSpacing:
+			values.letter_spacing(ComputeLength(p, font_size, document_font_size, dp_ratio, vp_dimensions));
```

Comment:
```suggestion
			values.letter_spacing(p->unit == Property::KEYWORD ? 0.f : ComputeLength(p, font_size, document_font_size, dp_ratio, vp_dimensions));
```
We only have a single keyword which computes to zero, otherwise resolve the length.